usr/share/rear/finalize/Linux-i386/670_run_efibootmgr.sh

```diff
@@ -19,60 +19,60 @@ is_true $EFI_STUB && return
 test -f "$TARGET_FS_ROOT/$UEFI_BOOTLOADER" || return 0
 
```

Comment:
Please add a
```
LogPrint "Creating EFI Boot Manager entries..."
```
here so that possibly subsequent messages appear in a meaningful context for the user
cf. finalize/Linux-i386/660_install_grub2.sh that also has such an initial `LogPrint`.

Reply:
Added.